pkgs/development/python-modules/cliff/default.nix

```diff
@@ -1,27 +1,31 @@
 { lib
-, buildPythonPackage
-, fetchPypi
 , autopage
+, buildPythonPackage
+, callPackage
 , cmd2
+, fetchPypi
 , importlib-metadata
 , installShellFiles
 , openstackdocstheme
 , pbr
 , prettytable
 , pyparsing
+, pythonOlder
 , pyyaml
-, stevedore
 , sphinx
-, callPackage
+, stevedore
 }:
 
 buildPythonPackage rec {
   pname = "cliff";
-  version = "4.2.0";
+  version = "4.3.0";
+  format = "setuptools";
```

Comment:
Whut!?

```
pythonOutputDistPhase
Executing pythonOutputDistPhase
The build contains no ./dist directory.
If this project is not setuptools-based, pass
  format = "other";
to buildPythonApplication/buildPythonPackage or another appropriate value as described here:
  https://nixos.org/manual/nixpkgs/stable/#buildpythonpackage-function
```

https://github.com/openstack/cliff/tree/4.3.0

Reply:
Locally on x86-64 I don't get this error. Not sure whatŝ going on here.

Reply:
Could it be actually coming from the tests here https://github.com/fabaff/nixpkgs/blob/cliff-bump/pkgs/development/python-modules/cliff/tests.nix#L10 ?